foodForChange.c

```diff
@@ -30,13 +30,15 @@ void foodForChange(){
 
     int recipesNumber;
     Recipe *recipes = readFile(&recipesNumber);
-    printf("Recipes: %d\n", recipesNumber);
+    printf("Loaded recipes: %d\n", recipesNumber);
 
     const int people = requestAmountOfPeople();
+    const int recipeIndex = requestRecipeName(recipes, recipesNumber);
+    printRecipe(recipes[recipeIndex], people);
```

Comment:
Shouldn't it be running after `makeListOfRecipes` and `printListOfRecipes`?

Reply:
yes you are very right